stl/inc/type_traits

```diff
@@ -1877,8 +1934,8 @@ public:
     }
 
     template <class... _Types>
-    auto operator()(_Types&&... _Args) const -> decltype(_STD invoke(get(), _STD forward<_Types>(_Args)...)) {
-        return _STD invoke(get(), _STD forward<_Types>(_Args)...);
+    auto operator()(_Types&&... _Args) const -> decltype(_STD invoke(get(), static_cast<_Types&&>(_Args)...)) {

```

Comment:
`reference_wrapper` seems `invoke`-ish but I could see an argument to restore `forward` here.

Reply:
This change is fine by me.